s3sync.go

```diff
@@ -129,12 +144,18 @@ func (m *Manager) Sync(source, dest string) error {
 		if err != nil {
 			return err
 		}
+		startTime = time.Now()
 		return m.syncLocalToS3(ctx, chJob, source, destS3Path)
 	}
 
 	return errors.New("local to local sync is not supported")
 }
 
+// GetStatistics returns the structure that contains the sync statistics
+func (m *Manager) GetStatistics() SyncStatistics {
+	return m.statistics
```

Comment:
I guess it may cause data race if it is called during sync.
```go
	return SyncStatistics{
		Bytes: atomic.LoadInt64(&m.statistics.Bytes),
		...
	}
```
would be safe.

Reply:
At the end I had to tweak things around to make it thread safe for reading. The problem was that I could not make `time.Duration` work with `atomic.AddInt64` or `atomic.AddPointer` and the alternative was using a `sync.RWLock` for all fields, for which I would still need the get convenience functions to ensure thread safety.

Let me know what you think, I'm open to suggestions to make it cleaner/better.

Reply:
acknowledged. I will change to `RWLock()`. Thanks for the input!

Reply:
Done.